src/main/kotlin/ch/hevs/cloudio2/cloud/services/ExecJobsService.kt

```diff
@@ -0,0 +1,62 @@
+package ch.hevs.cloudio2.cloud.services
```

Comment:
This service is a little pointless or am I wrong?

Reply:
For now yes, since I'm still working on the jobsService it does nothing